likecoin/admin/restful.php

```diff
@@ -45,6 +45,167 @@ function likecoin_rest_refresh_publish_status( $request ) {
 	$data         = likecoin_parse_publish_status( $matters_info );
 	return new WP_REST_Response( $data, 200 );
 }
+/**
+ * Add refresh publish status endpoint
+ *
+ * @param object| $post WordPress post object.
+ */
+function likecoin_get_post_image_url( $post ) {
+	$urls                  = array();
+	$content               = apply_filters( 'the_content', $post->post_content );
+	$dom_document          = new DOMDocument();
+	$libxml_previous_state = libxml_use_internal_errors( true );
+	$dom_content           = $dom_document->loadHTML( '<template>' . mb_convert_encoding( $content, 'HTML-ENTITIES' ) . '</template>', LIBXML_HTML_NOIMPLIED | LIBXML_HTML_NODEFDTD );
```

Comment:
```suggestion
	$libxml_previous_state = libxml_use_internal_errors( true );
	$dom_content           = $dom_document->loadHTML( '<template>' . mb_convert_encoding( $content, 'HTML-ENTITIES' ) . '</template>', LIBXML_HTML_NOIMPLIED | LIBXML_HTML_NODEFDTD );
	libxml_clear_errors();
	libxml_use_internal_errors( $libxml_previous_state );
```
$libxml_previous_state is used for recovering after parsing html